app/src/ereferrals/java/org/eyeseetea/malariacare/data/sync/importer/strategies/PullControllerStrategy.java

```diff
@@ -199,9 +203,12 @@ public void downloadAsyncLanguagesFromServer() throws Exception {
             CredentialsReader credentialsReader = CredentialsReader.getInstance();
             IConnectivityManager connectivity = NetworkManagerFactory.getConnectivityManager(
                     PreferencesState.getInstance().getContext());
+        ISettingsRepository settingsRepository = new SettingsDataSource(
+                PreferencesState.getInstance().getContext());
 
             DownloadLanguageTranslationUseCase downloader =
-                    new DownloadLanguageTranslationUseCase(credentialsReader, connectivity);
+                    new DownloadLanguageTranslationUseCase(credentialsReader, connectivity,
```

Comment:
Pass settingsRepository as argument to DownloadLanguageTranslationUseCase is not necessary